src/components/tx-flow/flows/RecoverAccount/RecoverAccountFlowReview.tsx

```diff
@@ -140,9 +140,7 @@ export function RecoverAccountFlowReview({ params }: { params: RecoverAccountFlo
         <RedefineBalanceChanges />
       </TxCard>
 
-      <TxCard>
```

Comment:
I tried this and while it works there is a slightly larger gap because `TxCard` itself is not empty but its child element `CardContent` while `TxCard` applies some margin.

<img width="731" alt="Screenshot 2024-07-29 at 19 47 39" src="https://github.com/user-attachments/assets/77274796-5f5c-4047-bff7-007175414a31">